src/app/components/buttons/BigButton/BigButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactNode } from "react";
+import "./BigButton.scss";
+
+const BigButton: React.FC<Props> = ({
```

Comment:
as said previously, BigButton should be a type of Button component which is choosen via some prop